core/src/test/java/org/jsmart/zerocode/core/runner/ZeroCodeMultiStepsScenarioRunner_FailedAssertions.java

```diff
@@ -0,0 +1,73 @@
+package org.jsmart.zerocode.core.runner;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.jsmart.zerocode.core.domain.reports.ZeroCodeReportProperties.TARGET_REPORT_DIR;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.jsmart.zerocode.core.di.provider.ObjectMapperProvider;
+import org.jsmart.zerocode.core.domain.reports.ZeroCodeReport;
+import org.jsmart.zerocode.core.runner.e2e.FailSimpleAssertionInMultiStep;
+import org.junit.Test;
+import org.junit.runner.JUnitCore;
+import org.junit.runner.Result;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class ZeroCodeMultiStepsScenarioRunner_FailedAssertions {
```

Comment:
Thanks for the test coverage.

Can you pls rename the class to (ending with *...Test*.java)?
> ZeroCodeMultiStepsScenarioRunnerFailedAssertionsTest

Otherwise it may not be the picked by CI build :-).

Reply:
My bad.. 👍